### PR TITLE
doing homework 20

### PR DESCRIPTION
**Pull request by Hromy-Ann**

### CODE REVIEW

File: src/main/java/com/practicesoftwaretesting/cart/asserts/UpdateCartResponseAsserts.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+import com.practicesoftwaretesting.cart.model.UpdateCartResponse;
+import lombok.AllArgsConstructor;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@AllArgsConstructor
+public class UpdateCartResponseAsserts {
+    private static final String EXPECTED_RESULT = "item added or updated";
+    private final UpdateCartResponse actual;
```
Comment (alexshamrai) [inline src/main/java/com/practicesoftwaretesting/cart/asserts/UpdateCartResponseAsserts.java:12]:
```suggestion

    private final UpdateCartResponse actual;
```

File: src/main/java/com/practicesoftwaretesting/cart/asserts/CreateCartResponseAsserts.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import com.practicesoftwaretesting.cart.model.CreateCartResponse;
+import lombok.AllArgsConstructor;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@AllArgsConstructor
+public class CreateCartResponseAsserts {
+    private final CreateCartResponse actual;
+
+    public CreateCartResponseAsserts hasSomeId() {
```
Comment (alexshamrai) [inline src/main/java/com/practicesoftwaretesting/cart/asserts/CreateCartResponseAsserts.java]:
```suggestion
    public CreateCartResponseAsserts idIsNotNull() {
```
or `hasValidId`

**Review by alexshamrai — CHANGES_REQUESTED** [review]

File: src/main/java/com/practicesoftwaretesting/cart/asserts/UpdateCartResponseAsserts.java (hunk 1 above)
Comment (Hromy-Ann) [reply src/main/java/com/practicesoftwaretesting/cart/asserts/UpdateCartResponseAsserts.java:12]: fixed

File: src/main/java/com/practicesoftwaretesting/cart/asserts/CreateCartResponseAsserts.java (hunk 2 above)
Comment (Hromy-Ann) [reply src/main/java/com/practicesoftwaretesting/cart/asserts/CreateCartResponseAsserts.java]: fixed

**Review by alexshamrai — APPROVED** [review]

**Pull request merged**